imports/src/parser/parse_package.rs

```diff
@@ -77,11 +77,13 @@ impl ImportParser<'_> {
         }
 
         // Get a vector of all packages in the source directory.
-        let entries = fs::read_dir(path)
+        let mut entries = fs::read_dir(path)
             .map_err(|error| ImportError::directory_error(error, &error_path, span))?
             .collect::<Result<Vec<_>, std::io::Error>>()
             .map_err(|error| ImportError::directory_error(error, &error_path, span))?;
 
+        entries.sort_by_key(|entry| entry.path());
```

Comment:
Can you add a comment explaining why this is done?

Reply:
Will do.